qutebrowser/browser/network/qutescheme.py

```diff
@@ -153,7 +153,7 @@ def qute_help(win_id, request):
             url=request.url().toDisplayString(),
             error="This most likely means the documentation was not generated "
                   "properly. If you are running qutebrowser from the git "
-                  "repository, please run scripts/asciidoc2html.py."
+                  "repository, please run scripts/asciidoc2html.py. "
```

Comment:
Seems like an unrelated change - is this actually fixing something or is it an accident?